### ISSUE TITLE
Get speaker posteriors from local EEND model

### ISSUE THREAD

**Issue by xiangzai0115** [FrenchKrab/IS2023-powerset-diarization#2]
Hi,

Thanks for this amazing work!
Is there any way to get speaker posteriors from local EEND model?

Cheers,
Xiang

**Comment by FrenchKrab** [comment]
Sure ! Take any example from the repository that uses the Inference class (eg [[1]](https://github.com/FrenchKrab/IS2023-powerset-diarization#using-checkpoints-for-the-segmentation-model-only), [[2]](https://github.com/FrenchKrab/IS2023-powerset-diarization/blob/master/notebook_ps_diarization.ipynb)), and add the option `skip_conversion=True`. For example:

```python
from pyannote.audio import Inference, Model

model = Model.from_pretrained("powerset_pretrained.ckpt")
inf = Inference(model, step=2.5, skip_conversion=True)
result = inf(my_audio_file)
result
```

This would give you something like that as output
![image](https://github.com/FrenchKrab/IS2023-powerset-diarization/assets/14005967/22a6d849-022d-4c52-83c8-f660f5025ce3)

Note that this is the output of the LogSoftmax layer. To obtain "probabilities":
```python
import numpy as np
result.data = np.exp(result.data)
result
```
![image](https://github.com/FrenchKrab/IS2023-powerset-diarization/assets/14005967/08720e5b-d943-4dad-b25a-725ba89fc22b)

Finally, if you want the logits, you can remove the LogSoftmax with
```python
model.activation = torch.nn.Identity()
```
and run the Inference as usual, with `skip_conversion=True`.

**Comment by xiangzai0115** [comment]
Cool, that's wonderful!!

Thanks!

**Comment by FrenchKrab** [comment]
I'm closing the issue for now, but please do reopen it if any details are missing or if you encounter problems.

**Comment by rohitpaturi** [comment]
I have a follow-up question regarding this. The inference pipeline seems to provide posteriors/decisions at the frame level. What is the frame resolution to convert the frame level decisions to time/temporal decisions?
